content/docs/0.8.1/volumes-and-nodes/expansion.md

```diff
@@ -3,24 +3,34 @@
   weight: 4
 ---
 
-## Overview
-- Longhorn supports OFFLINE volume expansion only. 
-- Longhorn will expand frontend (e.g. block device) then expand filesystem.
+Volumes are expanded in two stages. First, Longhorn expands the frontend (the block device), then it expands the filesystem.
+
+To prevent the frontend expansion from being interfered by unexpected data R/W, Longhorn supports offline expansion only.  The `detached` volume will be automatically attached to a random node with [maintenance mode.](../../concepts/#maintenance-mode)
+
+Rebuilding and adding replicas is not allowed during the expansion, and expansion is not allowed while replicas are rebuilding or being added.
+
+The Longhorn volume size will be updated after the expansion, but the capacity of corresponding PVC and PV won't change. You will need to update the PV and PVC Kubernetes resources separately.
```

Comment:
`If the volume was not expanded though CSI interface (e.g. for Kubernetes older than v1.16), the capacity of corresponding PVC and PV won't change.`

Reply:
replaced the sentence

Reply:
updated